src/amo/components/InstallButtonWrapper/index.js

```diff
@@ -99,32 +101,43 @@ export const InstallButtonWrapperBase = (props: InternalProps) => {
           'InstallButtonWrapper--notFirefox': !isFirefox({ userAgentInfo }),
         })}
       >
-        <AMInstallButton
-          addon={addon}
-          canUninstall={canUninstall}
-          className={makeClassName(
-            className ? `AMInstallButton--${className}` : '',
-            {
-              'AMInstallButton--noDownloadLink': !showDownloadLink,
-            },
-          )}
-          currentVersion={currentVersion}
-          defaultButtonText={defaultButtonText}
-          disabled={!isCompatible}
-          enable={enable}
-          hasAddonManager={hasAddonManager}
-          install={install}
-          isAddonEnabled={isAddonEnabled}
-          puffy={puffy}
-          setCurrentStatus={setCurrentStatus}
-          status={installStatus}
-          uninstall={uninstall}
-        />
-        <GetFirefoxButton
-          addon={addon}
-          buttonType={getFirefoxButtonType}
-          className={className ? `GetFirefoxButton--${className}` : ''}
-        />
+        {showLinkInsteadOfButton && (
```

Comment:
Instead of two statements, one where `showLinkInsteadOfButton` is `true` and one where `showLinkInsteadOfButton` is `false`, you could use a ternary operator, e.g.,

```
{showLinkInsteadOfButton ? (
  <a className={makeClassName('oldDownload')} href={installURL}>
    {i18n.gettext('Download Older Version')}
  </a>) : (<><AMInstallButton ...)
}
```